src/stdatamodels/jwst/datamodels/tests/test_wcs.py

```diff
@@ -15,54 +15,62 @@ def test_get_fits_wcs(tmpdir):
     with ImageModel(FITS_FILE) as dm:
         # Refer to the data array to initialize it.
         dm.data = np.zeros((5, 5))
+        with pytest.warns(DeprecationWarning):
+            wcs1 = dm.get_fits_wcs()
 
-        # Now continue with the test.
-        wcs1 = dm.get_fits_wcs()
-        dm2 = dm.copy()
-        wcs2 = dm2.get_fits_wcs()
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Comment:
```suggestion
    with warnings.catch_warnings():
        warnings.filterwarnings("ignore", category=DeprecationWarning)
```
Instead of this generic warning filter would you wrap only the `get_fits_wcs` and `set_fits_wcs` calls in `pytest.warns` contexts with `match` regexes.

Alternatively, if this test will be removed when `get_fits_wcs` is removed the whole test can be wrapped with [pytest.mark.filterwarnings](https://docs.pytest.org/en/6.2.x/warnings.html#pytest-mark-filterwarnings) with the corresponding filter string to ignore only the new deprecation warnings.

Reply:
see https://github.com/spacetelescope/stdatamodels/pull/482/commits/dea4c3a1d66c5610a100e4dbff46dedd60c7961b hopefully that does what you're asking?